Makefile

```diff
@@ -11,8 +11,6 @@ GOCACHE ?= $(shell echo ${PWD})/$(OUTPUT_DIR)/gocache
 GOARCH ?= amd64
 # golang global flags
 GO_FLAGS ?= -v -mod=vendor
-# golang test floags
-GO_TEST_FLAGS ?= -failfast -timeout=25m
```

Comment:
can we keep the same `failfast` behaviour?, I guess ginkgo have flags for it.

Reply:
Correct, ginkgo has `--failFast`. I'll add it and retain it to be configurable with enabled as default. I think that when locally running tests as a dev, having the test to stop immediately is probably expected to start investigating that failure right away. In a CD pipeline I would expect all tests to always run to see that maybe the buildah tests all fail while kaniko and buildpack continue to run.

Reply:
Discussed with @qu1queee; we consolidate the ginkgo flags to a single environment variable.

Reply:
Done, see https://github.com/redhat-developer/build/pull/163/commits/7ee9f0765f5f92753d685a0c5ca7efed0be5665d#diff-b67911656ef5d18c4ae36cb6741b7965R29